k8s/releases/pastebin/pastebin.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pastebin
+  namespace: pastebin-prod
+  labels:
+    app: pastebin
+  annotations:
+    fluxcd.io/automated: "true"
+    filter.fluxcd.io/chart-image: regex:^(v[0-9]+.[0-9]+.[0-9]+)$
+spec:
+  releaseName: pastebin
+  chart:
+    repository: https://mozilla-it.github.io/helm-charts/
+    name: pastebin
+    version: "0.1.1"
+  values:
+    configMap:
+      data:
+        DB_PORT: "3306"
+        VAL: "5"
+    deployment:
+      port: "8000"
+      replicaCount: "1"
+    externalSecrets:
+      enabled: true
+      name: pastebin
+      secrets:
+      - key: /prod/pastebin/envvar
+        name: DB_HOST
+        property: database_host
+      - key: /prod/pastebin/envvar
+        name: DB_NAME
+        property: database_name
+      - key: /prod/pastebin/envvar
+        name: DB_PASS
+        property: database_password
+      - key: /prod/pastebin/envvar
+        name: DB_USER
+        property: database_user
+      - key: /prod/pastebin/envvar
+        name: SESSION_KEY
+        property: session_key
+    image:
+      pullPolicy: Always
+      tag: v3.5.4
+    ingress:
+      hosts:
+        - host: paste.prod.mozit.cloud
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+              serviceName: pastebin
+              servicePort: 80
+        - host: paste.mozilla.org
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+              serviceName: pastebin
+              servicePort: 80
+        - host: pastebin.mozilla.org
+          paths:
+            - path: /
+              pathType: ImplementationSpecific
+              serviceName: pastebin
+              servicePort: 80
+      le:
```

Comment:
Hopefully last comment for me, sorry I didn't see this before. What does `le` mean here haha. I'm having a hard time tracking that down

Reply:
It's short for letsencrypt, I think, but I'd like to know where it gets pulled in from what module, too.

Reply:
@bkochendorfer @floatingatoll its referenced here: https://github.com/mozilla-it/helm-charts/blob/f5b8d634f032730cfd90f598bc4accaa514250bf/charts/pastebin/values.yaml#L69

Reply:
e.g. this is a helmrelease resources that uses that pastebin helm chart & provides those values to the helm chart, so any questions on those values / what they're used for is answered by the helm chart.

Reply:
Going even deeper, this is where that helm chart uses this value: https://github.com/mozilla-it/helm-charts/blob/f5b8d634f032730cfd90f598bc4accaa514250bf/charts/pastebin/templates/issuer.yaml#L9 basically, it determines if we use prod or stage lets encrypt for the certificate issuer.

Reply:
Nice thanks for the info @cmharlow !